apps/studio/components/interfaces/Organization/Usage/Usage.constants.tsx

```diff
@@ -109,7 +109,7 @@ export const USAGE_CATEGORIES: (subscription?: OrgSubscription) => CategoryMeta[
             chartPrefix: 'Average',
             unit: 'bytes',
             description:
-              'Database size refers to the monthly average database space usage, as reported by Postgres. Paid Plans use auto-scaling disks.\nBilling is based on the average daily database size used in GB throughout the billing period. Billing is independent of the provisioned disk size.',
+              'Database size refers to the actual amount of space used by all your database objects, as reported by Postgres. Paid Plans use auto-scaling disks and are billed on allocated disk instead of database size.',
```

Comment:
Since this is relevant for Free plans only, maybe we could drop the second sentence regarding auto-scaling?
Of course it's good to know the difference between Free Plan and paid plans regarding database size and disk size, but not sure if that might confuse users rather than help them.